tests/e2e/support.py

```diff
@@ -48,7 +67,7 @@ def create_namespace_with_name(self, namespace_name):
         )
         self.api_instance.create_namespace(namespace_body)
     except Exception as e:
-        return _kube_api_error_handling(e)
+        print(f"Error creating namespace with name : {e}")
```

Comment:
@sutaakar  Actually, the kube_api_error_handling method was being imported from codeflare-sdk package, if we want to use it in case of disconnected then need to add codeflare-sdk in mnist job's requirements.txt which doesn't seems necessary, WDYT?

Reply:
Would it be possible to produce error instead of logging it only?
If I get it right the test will continue running and will fail on some other place, right?

Reply:
Yes, right! It will keep executing even if namespace creation failed..
Thinking what to use here to stop execution once failed :thinking: